src/renderer/components/+extensions/extensions.tsx

```diff
@@ -187,15 +187,17 @@ export class Extensions extends React.Component {
     await Promise.all(
       requests
         .filter(req => !req.data && req.filePath)
-        .map(request => {
-          return fse.readFile(request.filePath).then(data => {
+        .map(async request => {
+          try {
+            const data = await fse.readFile(request.filePath);
```

Comment:
Does this change anything else than syntax?

Reply:
No. Initially I noticed that previous implementation was slow on Windows and `readFileSync` was significantly faster. Then I just converted it to use `readFile` with await and did not see any effect. Perhaps the original issue with slowness was something else.